packages/react-core/src/components/Button/examples/ButtonAriaDisabled.tsx

```diff
@@ -1,11 +1,12 @@
 import React from 'react';
-import { Button } from '@patternfly/react-core';
+import { Button, Tooltip } from '@patternfly/react-core';
 import TimesIcon from '@patternfly/react-icons/dist/esm/icons/times-icon';
 import PlusCircleIcon from '@patternfly/react-icons/dist/esm/icons/plus-circle-icon';
 
 export const ButtonAriaDisabled: React.FunctionComponent = () => (
   <React.Fragment>
-    <Button isAriaDisabled>Primary aria disabled</Button> <Button isAriaDisabled>Secondary aria disabled</Button>{' '}
+    <Button isAriaDisabled>Primary aria disabled</Button>{' '}
```

Comment:
Once this lint error is resolved we can take it in.

Reply:
For some reason, the lint error is simply the new `{' '}` I added to give spacing between the examples. If I get rid of it, there are no issues. But it makes two of the buttons right next to each other

Reply:
hmmm I am not sure why it is complaining about this line in particular since we have  `{' '}` elsewhere.